day1/Readme.md

```diff
@@ -1,2 +1,2 @@
 # Report Here
-
+https://github.com/anton-maslakou/zabbix-tasks/blob/day1/day1/wordfile/report-zabbix.docx
```

Comment:
Check [this](https://en.wikipedia.org/wiki/Markdown) and try [this](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)